appium/lib/screen-object.rb

```diff
@@ -69,81 +66,80 @@ def enter
     #pending implementation
```

Comment:
This enter one has pending implementation but can be easily fixed. Can you add `send_keys(:enter)` OR ``adb shell input keyevent 4`` and it should be good.